src/extension.ts

```diff
@@ -196,19 +195,19 @@ function initializeIntervalJobs() {
             if (!updatedMilestonesDb) {
                 pushUpdatedMilestones();
             }
-            fetchMilestonesForYesterdayAndToday();
+            fetchMilestones();
 
             // summary
             pushSummaryToDb();
         }
-    }, one_min_millis * 1);
+    }, one_min_millis * 5);
 
     one_hour_interval = setInterval(() => {
         if (checkIfNameChanged()) {
             logOut();
         } else {
             if (updatedMilestonesDb && sentMilestonesDb) {
-                fetchAllMilestones();
+                fetchMilestones();
```

Comment:
why do we need to fetch after we've sent or updated? Shouldn't we locally update the milestones?

Reply:
Not sure but this check is only hourly so I'll leave it for now